.github/workflows/distro-release.yml

```diff
@@ -0,0 +1,125 @@
+name: Windows Release by Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        description: 'Tag to build Windows Release from'
+        required: true
+
+jobs:
+  config:
+    runs-on: aws-micro
+    outputs:
+      app_version: ${{ steps.set.outputs.app_version }}
+      release_tag: ${{ steps.set.outputs.release_tag }}
+      windows_x64_config_matrix: ${{ steps.set.outputs.matrix }}
+      full_config_build: "false"
+      internal_build: "false"
+      upload_test_artifacts: "false"
+      nuget_build: "false"
+    steps:
+      - name: Set release config
+        id: set
+        run: |
+          TAG="${{ github.event.inputs.release_tag || github.ref_name }}"
+          APP_VERSION="${TAG#v}" # strip leading v
+          echo "app_version=$APP_VERSION" >> $GITHUB_OUTPUT
+          echo "release_tag=$TAG" >> $GITHUB_OUTPUT
+
+          # Minimal matrix: Release only
+          MATRIX=$(jq -c . < .github/workflows/matrix/windows-release-config.json)
+          echo "matrix=$MATRIX" >> $GITHUB_OUTPUT
+
+          jq \
+            --arg vs19 "${{ needs.Configuration.outputs.vs19_vcpkg_version }}" \
+            --arg vs22 "${{ needs.Configuration.outputs.vs22_vcpkg_version }}" \
+            '(.include[] | select(.["vcpkg-version"]=="__VCPKG_VERSION_VS19__") | .["vcpkg-version"]) = $vs19
+            | (.include[] | select(.["vcpkg-version"]=="__VCPKG_VERSION_VS22__") | .["vcpkg-version"]) = $vs22' \
+            "$MATRIX_FILE" > tmp.json
+
+          echo "matrix=$(jq -c . tmp.json)" >> $GITHUB_OUTPUT
+
+  windows-build-test:
+    needs: [ config ]
+    uses: ./.github/workflows/build-test-windows.yml
+    with:
+      config_matrix:         ${{ needs.config.outputs.windows_x64_config_matrix }}
+      full_config_build:     ${{ needs.config.outputs.full_config_build == 'true' }}
+      internal_build:        ${{ needs.config.outputs.internal_build == 'true' }}
+      upload_test_artifacts: ${{ needs.config.outputs.upload_test_artifacts == 'true' }}
+      nuget_build:           ${{ needs.config.outputs.nuget_build }}
+    secrets: inherit
+
+  distribute-windows:
+    needs:
+      - config
+      - windows-build-test
+    uses: ./.github/workflows/distribute.yml
+    with:
+      app_version: ${{ needs.config-release.outputs.app_version }}
+      release_tag: ${{ needs.config-release.outputs.release_tag }}
+    secrets:
+      BUILD_MACHINE_TOKEN: ${{ secrets.BUILD_MACHINE_TOKEN }}
+
+  update-win-version:
+    if: ${{ needs.config.outputs.upload_artifacts == 'true' && needs.config.outputs.build_enable_windows == 'true' }}
+    needs:
+      - config
+      - windows-build-test
+    uses: ./.github/workflows/update-win-version.yml
+    with:
+      vs19_vcpkg_version: ${{ needs.config.outputs.vs19_vcpkg_version }}
+    secrets: inherit
+
+  upload-distributions:
+    if: ${{ !cancelled() && needs.config.outputs.upload_artifacts == 'true' }}
```

Comment:
can it be active? as i see we only can start this workflow with manual dispatch and `upload_artifacts` disabled?

Reply:
rm